vanilla-framework/vanilla-framework
# 1878 code snippet

## Done

- Update the code icon to a $
- Fix the copy button styling

## QA

- Pull code
- Run `./run serve --watch`
- Open the [Code copyable pattern](http://0.0.0.0:8101/vanilla-framework/examples/patterns/code-copyable/) in your browser
- See that it matches the [design](https://user-images.githubusercontent.com/17748020/41605000-6731bae4-73d8-11e8-9477-faee07f72304.png)

## Details

Fixes #1878
